### DISCUSSION TITLE
how to prevent reline for multi command within one line

### DISCUSSION THREAD

**Discussion by heartacker** [mvdan/sh#1048] [Q&A]
if we use sh to format the command
```
devmem 0x90415A00 32 0xA5A5A5A5;devmem 0x90415A00 32 0x5A5A7676
```
we get this
```
devmem 0x90415A00 32 0xA5A5A5A5
devmem 0x90415A00 32 0x5A5A7676
```

how to keep my code in one line

**Comment by mvdan** [comment]
See https://github.com/mvdan/sh/issues/261 which is closed. We only plan to allow this in a few select cases like https://github.com/mvdan/sh/issues/564 or https://github.com/mvdan/sh/issues/679.
